bin/docker-assets-precompile

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# For building production docker image
+#
+# It sets the envs inside the docker image for precompiling the assets
+# Because to precompile the assets, Rails initializes the app.
+# And it requires the envs as we always use `ENV.fetch` to setup the variables
+#
+# Related issue: https://github.com/rails/rails/issues/32947
+# TODO: https://github.com/nimblehq/rails-templates/issues/326
+
+require 'yaml'
+
+rails_env = ENV.fetch('RAILS_ENV', 'production')
+
+if rails_env == 'production'
+  # Read the contents of the .env file
+  File.readlines('.env.development.local.example').each do |line|
```

Comment:
If we build the Docker image, why not using the attribute `env_file`: https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute

Reply:
I tried the env_file but not sure why the ENV can't find there...still need time to figure out @olivierobert :D